ios/FluentUI/Tab Bar/SideTabBar.swift

```diff
@@ -39,6 +39,8 @@ open class SideTabBar: UIView {
             if let avatarView = avatarView {
                 avatarView.avatarSize = .medium
                 avatarView.translatesAutoresizingMaskIntoConstraints = false
+                avatarView.overrideAccessibilityLabel = "Accessibility.LargeTitle.ProfileView".localized
+                avatarView.accessibilityTraits = .button
```

Comment:
should it be button trait only if the client is actually going to handle the tap on avatarview?

Reply:
I mentioned in the description that there's an assumption that the avatar view will always be used as a button. I wonder if a client would ever want a different behavior. 
Here are 3 options:
- Always assume that it will be a button. From a design perspective, this might make sense since we don't expect anyone to show the avatar view if it can't be interacted with.
- Only set the trait as button if the client provides a handler for the tap gesture recognizer. Otherwise, default to image.
- Never set the trait. Since the AvatarView is a public property, the clients can set the AX trait themselves.

What do you prefer?

Reply:
I vote for option #2

"Only set the trait as button if the client provides a handler for the tap gesture recognizer. Otherwise, default to image."

Reply:
There is no way, in swift, to check if a specific optional method is implemented. The best I can do here is check if the delegate is nil or not which isn't great.